app/controllers/placements/support/organisations/users_controller.rb

```diff
@@ -26,8 +25,26 @@ def create
     end
   end
 
+  def remove; end
+
+  def destroy
+    membership = @user.memberships.find_by(organisation: @organisation)
+    if membership.present? && membership.destroy
+      RemoveUserService.call(@user, @organisation)
```

Comment:
This is not a change request – just an opinion and some thinking out loud.

It feels weird (slightly pointless?) to have an entire service object simply as a wrapper around sending the notification email. The service only does one thing so it just adds a layer of abstraction which you might consider unnecessary.

But I appreciate that this is a continuation of an earlier conversation with @Nitemaeric about where to locate this sort of business logic: https://github.com/DFE-Digital/itt-mentor-services/pull/142#discussion_r1461766399

I'm wondering whether it'd be appropriate to move the database changes into the service object too? I could imagine the service object accepting a `Membership` or a `User` + `Organisation` and performing the destroy and sending the email notification as one action. That seems like an appropriate coupling to me because it describes an atomic interaction/transaction in terms of business logic.